src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/DataSetViewModels.cs

```diff
@@ -36,3 +38,27 @@ public record DataSetLiveVersionViewModel : DataSetVersionViewModel
 {
     public required DateTimeOffset Published { get; init; }
 }
+
+public record DataSetVersionSummaryViewModel
```

Comment:
I think it would be helpful to add the data set version id to this view model, even if it's not necessary for the FE.

Reply:
will do!